qp_fastp_minimap2/qp_fastp_minimap2.py

```diff
@@ -13,12 +13,15 @@
 from itertools import zip_longest
 
 from qiita_client import ArtifactInfo
-from qiita_client.util import system_call
 
+MEMORY = '16g'
```

Comment:
Someday we should consider putting these parameters in a config file or in the database.